src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -172,6 +190,9 @@ def get_attrs(
             if item.owner_id is not None:
                 owners_by_type[actor_type_to_string(item.owner.type)].append(item.owner_id)
 
+            activations = activations_by_alert_rule_id.get(item.id, [])
+            result[item]["activations"] = serialize(activations, **kwargs)
```

Comment:
We often like to batch serializations to avoid n+1s. Your serializer doesn't make db queries so it's safe for now, but if we batch them now then we don't have to worry about unintentionally adding them in the future